core/src/main/java/org/lflang/federated/launcher/FedLauncherGenerator.java

```diff
@@ -401,7 +401,7 @@ private String getRemoteLaunchCode(
         "        echo \"The source code can be found in org.lflang/src/lib/core/federated/RTI\"",
         "        exit 1",
         "    fi",
-        "    " + rtiLaunchString + " 2>&1 | tee -a " + logFileName + "' &",
+        "    " + rtiLaunchString + "\n 2>&1 | tee -a " + logFileName + "' &",
```

Comment:
This looks a little weird to me. Why is it reasonable to assume that the argument `rtiLaunchString` would have a trailing slash? Seems more like a problem that needs to be solved at the call site(s) of `getRemoteLaunchCode`. That said, I don't think this code is very critical, so let's just merge and move on...